src/Uno.UI/UI/Xaml/Markup/Reader/XamlConstants.cs

```diff
@@ -12,7 +12,7 @@ internal static class XamlConstants
 		public const string BundleResourcePrefix = "ms-appx:///";
 
 		public const string RootFoundationNamespace = "Windows.Foundation";
-		public const string RootWUINamespace = "Windows.UI";
+		public const string RootWUINamespace = "Windows" + ".UI"; // Keep split for the WinUI conversion tool
 		public const string RootMUINamespace = "Windows.UI";
```

Comment:
The name for this one is quite confusing.

`MUI` seems to mean "Microsoft.UI", but that's not the case?